### ISSUE TITLE
bCNC can add a simple mode of operation

### ISSUE THREAD

**Issue by viewsat** [vlachoudis/bCNC#533]
Recommended bCNC can add a simple mode of operation, on the right only show send G-code, like the following picture.
![bcnc1](https://cloud.githubusercontent.com/assets/15762267/23595559/bf4bce6a-025d-11e7-9942-2cde15c31c52.jpg)

**Comment by vlachoudis** [comment]
It could be a possible enhancement. You have anyhow this information in the "Terminal' which is synchronized with the replies of grbl. Not in a table format.

**Comment by kworsfold** [comment]
Agreed.... This would help in debugging as it would show the current line when it errors

**Comment by vlachoudis** [comment]
Actually in my TODO list was to make a step by step debugger. That you go in the editor and sending individual commands or blocks to grbl, and inspecting the state.... I need to find some time to do it.

**Comment by kworsfold** [comment]
That would be cool.... very useful making problems easier to resolve
